### DISCUSSION TITLE
Website down while domain name is being transferred - now back online

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#181] [Announcements]
"cosmology.info" and "cosmologystatement.org" are unavailable while the domain names are being transferred from HostMonster to Web Hosting Canada.  Once the transfer is done, some adjustments will have to be done to point the domain names back to GitHub.
This does not affect the discussion forum.

**Comment by RedshiftDrift** [comment]
Website transfer and DNS check done, ***A Cosmology Group*** is back on [cosmology.info](https://cosmology.info) and [cosmologystatement.org](https://cosmologystatement.org).

**Comment by mikehelland** [comment]
FWIW, your email "redshift@cosmology.info" bounced back. Was asking if there's something funky going on with this discussion group. Hit a few 404's today.

**Reply by RedshiftDrift** [reply]
*redshift @* *cosmology.info* is no longer active. There should not be any mention of it anywhere.

The active email address is given here: [https://cosmology.info/org/acg-organization.html#contacts](https://cosmology.info/org/acg-organization.html#contacts)

**Reply by mikehelland** [reply]
Thanks, that email is listed here:

https://github.com/a-cosmology-group

**Reply by RedshiftDrift** [reply]
Thanks for pointing that out, I corrected it.
